homeassistant/components/arcam_fmj/media_player.py

```diff
@@ -71,13 +59,12 @@ class ArcamFmj(MediaPlayerEntity):
     """Representation of a media device."""
 
     def __init__(
-        self, state: State, uuid: str, name: str, turn_on: Optional[ConfigType]
+        self, state: State, uuid: str,
     ):
         """Initialize device."""
         self._state = state
+        self._name = f"{DEFAULT_NAME} ({self._state.client.host}) - Zone: {state.zn}"
```

Comment:
Would it make sense to use the entry title as the first part, since that's the same ?

Reply:
Yes, nice idea. I let the device name also inherit from config entry. Two small issues with it that i think is okey for now:
* If config entry is renamed, the effect on device name and entity name won't occur until restart
* if device is renamed, it won't be inherited to entities now.